tests/js/user-tests.js

```diff
@@ -272,6 +272,24 @@ module.exports = {
     TestCase.assertEqual(err.message, "already confirmed", "User should already be confirmed.");
   },
 
+  async testResetPassword() {
+    let app = new Realm.App(appConfig);
+
+    const validEmail = randomVerifiableEmail();
+    const validPassword = "password123456";
+    await app.emailPasswordAuth.registerUser({ email: validEmail, password: validPassword });
+
+    const newPassword = "realm_tests_do_reset654321";
+    await app.emailPasswordAuth.callResetPasswordFunction({ email: validEmail, password: newPassword });
```

Comment:
Would it be worthing adding another test which calls this with the optional extra `...args`? e.g. 

```
await app.emailPasswordAuth.callResetPasswordFunction({ email: validEmail, password: newPassword }, "test", 123);
```

I'm not sure if/how we could verify that the args are actually being passed to the function, but it would check that we don't get an error calling it at least

Reply:
I guess the function could send the args back in the resolved promise?

Reply:
Yeah that's a good idea – I actually have no idea how the Realm app for these tests comes into being (e.g. where the template which contains the password reset function is)!